src/client/sync/rtu.rs

```diff
@@ -18,10 +17,12 @@ pub fn connect_slave(
     settings: &SerialPortSettings,
     slave: Slave,
 ) -> Result<Context> {
-    let mut core = Core::new()?;
-    let handle = core.handle();
-    let serial = Serial::from_path_with_handle(tty_path, settings, &handle.new_tokio_handle())?;
-    let async_ctx = core.run(async_connect_slave(&handle, serial, slave))?;
-    let sync_ctx = Context { core, async_ctx };
+    let mut rt = tokio::runtime::Runtime::new().unwrap();
```

Comment:
Why not map the error instead of `unwrap`?

Reply:
fixed